.github/workflows/deploy-frontends-to-production.yml

```diff
@@ -159,7 +159,14 @@ jobs:
       - name: Build and Push Docker Image
         run: |
           cd website/
-          docker build --build-arg REACT_WEB_STATIC_HOST=${{ secrets.REACT_WEB_STATIC_HOST }} --target=deployment --tag ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-website:${{ needs.image-tag.outputs.build_id }} -f docker/Dockerfile .
+          docker build \
+          --target=deployment \
+          --build-arg REACT_WEB_STATIC_HOST=${{ secrets.REACT_WEB_STATIC_HOST }} \
+          --build-arg REACT_NETMANAGER_BASE_URL=${{ secrets.REACT_NETMANAGER_BASE_URL_PROD }} \
```

Comment:
@ja-odur , this is for the production link as required.

Reply:
@ja-odur  and @Baalmart please help @123MwanjeMike add the new variables to the repo secrets

Reply:
I am not sure about this procedure for updating the secrets of this particular project. Perhaps @ja-odur could assist. cc @123MwanjeMike